src/main/scala/is/hail/utils/richUtils/RichContextIterator.scala

```diff
@@ -0,0 +1,11 @@
+package is.hail.utils.richUtils
+
+import is.hail.utils.WithContext
+
+class RichContextIterator[T](val i: Iterator[WithContext[T]]) {
+  def mapLines[U](f: T => U): Iterator[U] = i.map(_.map(f).value)
```

Comment:
There are other places in the test suites where this could be used:
- InfoScoreSuite
- InbreedingCoefficientSuite
- OrderedRDDSuite
- ConcordanceSuite
- BaldingNicholsSuite
- ImputeSexSuite